public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -32,55 +33,67 @@ import { LokiDatasource } from './datasource';
 import { getStreamSelectorPositions, NodePosition } from './modifyQuery';
 import { LokiQuery, LokiQueryType } from './types';
 
-export function formatQuery(selector: string | undefined): string {
```

Comment:
Removed this because:
- It doesn't format anything
- It trims the "selector" or returns an empty string
- Triming the query does not affect the only usage of this function